.env.docker

```
@@ -508,12 +508,44 @@ FORGOT_PASSWORD_URL='https://app.gauzy.co/#/auth/request-password'
+GAUZY_MCP_DEBUG=false
+MCP_APP_ID=co.gauzy.mcp-server
 MCP_APP_NAME="Gauzy MCP Server"
-MCP_APP_ID="co.gauzy.mcp-server"
-GAUZY_AUTO_LOGIN=true
-GAUZY_AUTH_EMAIL=default_email:employee@ever.co,admin@ever.co
-GAUZY_AUTH_PASSWORD=<your-password>
+NODE_ENV=development
+
+# MCP Transport Configuration
+# Options: stdio | http | websocket
+MCP_SERVER_MODE=stdio # Internal server runtime mode (e.g., stdio for Claude Desktop)
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)